src/taco-remote/lib/server.ts

```diff
@@ -147,7 +147,9 @@ class Server implements RemoteBuild.IServerModule {
         var buildInfo = this.buildManager.getBuildInfo(req.params.id);
         if (buildInfo) {
             res.set("Content-Type", "text/plain");
+            /* tslint:disable:no-bitwise */
             this.buildManager.downloadBuildLog(req.params.id, req.query.offset | 0, res);
```

Comment:
We can remove this |0 by replacing it with an explicit conversion to integer; probably parseInt would be best here